modules/virt-vm-storage-volume-types.adoc

```diff
@@ -5,19 +5,19 @@
 [id="virt-vm-storage-volume-types_{context}"]
 = Virtual machine storage volume types
 
-Virtual machine storage volume types are listed, as well as domain and volume settings.
+|===
+|Storage volume type |Description
 
-[horizontal]
-*ephemeral*::
-A local copy-on-write (COW) image that uses a network volume as a
+|ephemeral
+|A local copy-on-write (COW) image that uses a network volume as a
 read-only backing store. The backing volume
 must be a *PersistentVolumeClaim*. The ephemeral image is created when
 the virtual machine starts and stores all writes locally. The ephemeral
 image is discarded when the virtual machine is stopped, restarted, or
 deleted. The backing volume (PVC) is not mutated in any way.
 
-*persistentVolumeClaim*::
-Attaches an available PV to a virtual machine. Attaching a PV allows for the
+|persistentVolumeClaim
+|Attaches an available PV to a virtual machine. Attaching a PV allows for the
 virtual machine data to persist between sessions.
 +
```

Comment:
These are rendering literally now that it's a table. Try an empty line between the paras. 
(5 instances)

Reply:
Hi @aburdenthehand I looked at the table which has 6 rows. Do we want to out a hard return after each paragraph in each row? If you could please explain, much appreciated because I did not understand the request. Thanks!

Reply:
@aburdenthehand We want an empty line after each paragraph in both columns?

Reply:
Referring to the '+' character on line 22 (and in 4 other lines). 
In the original this meant the description was split into separate paras:
https://docs.openshift.com/container-platform/4.5/virt/virtual_machines/virt-create-vms.html#virt-vm-storage-volume-types_virt-create-vms

But now they're rendering literally, and it's not separating the content into separate paras:
https://bug-1858756-vol-type-table--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html#virt-vm-storage-volume-types_virt-create-vms

Reply:
Thanks for the clarification. Adding the empty lines now displays the space between the paragraphs.